config/thinking_sphinx.yml

```diff
@@ -62,4 +70,5 @@ test: &test
 
 production:
   <<: *default
+  skip_running_check: true # daemon is remote in production
```

Comment:
This is to allow us running `ts:index` without funny workarounds. I spotted this just now :face_with_head_bandage: